project/Dependencies.scala

```diff
@@ -30,19 +30,22 @@ object Dependencies {
     val scalaMockito = "1.17.12"
     val scalaLangJava8Compat = "1.0.2"
 
-    val jacksonModuleScala = "2.13.1"
+    val jacksonModuleScala = "2.14.2"
 
     val specs2 = "4.10.0"
     val typesafeConfig = "1.4.2"
 
-    val spring = "2.6.1"
+    val spring = "3.1.0"
```

Comment:
I think this version can't be used due to incompatibility with our target Java version.